src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests.csproj

```diff
@@ -25,6 +25,7 @@
     <ProjectReference Include="..\DeployTestDependencies\DeployTestDependencies.csproj">
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.VisualStudio.ProjectSystem.CSharp\Microsoft.VisualStudio.ProjectSystem.CSharp.csproj" />
```

Comment:
Is this right? Seems like a layer violation - managed.unittests shouldn't know about C#